src/main/drivers/nvic.h

```diff
@@ -54,6 +54,9 @@
 #define NVIC_PRIORITY_SUB(prio) (((prio)&(0x0f>>(7-(NVIC_PRIORITY_GROUPING))))>>4)
 #else
 // utility macros to join/split priority
+#if defined(UNIT_TEST) && !defined(NVIC_PriorityGroup_2)
```

Comment:
Rather than have code conditional on `UNIT_TEST` define `NVIC_PriorityGroup_2` in `src/test/unit/platform.h`